cli/example-ifc.config.js

```diff
@@ -13,14 +13,15 @@ module.exports = function(frameRouter) {
     {
       locale: 'en-US',
       hostRootUrl: window.location.origin,
+      registeredKeys: ['a', 'b', 'actrl', 'ashift', 'actrlshift'],
```

Comment:
I'm not a huge fan of this key format.  Any reason we can't do something like `ctrl+shift+a`?  That seems much more intuitive/more consistent with other programs that do key bindings.

Looking forward to other parts of the review, it seems like if I registered `aaltctrl` that would just silently fail, right?  We might want to do some sort of parsing/verification on these keys.

Reply:
Yeah I could change the format and add validation. The other part that is not great currently is ordering matters. `aaltctrl` is not `ctrlalta`. Really I do not like having this list of string in general and maybe the solution is using the Key class as the list but I am not sure how that would work for our consumers.

Reply:
Yeah, that was part of the reason I thought something like `+` delimited lists made more sense.  It seems like it would be fairly easy `.split('+')` it and do simple validation/order independent parsing

Reply:
you would have to worry about using + as your key with that approach :)

But i agree this format should change.